packages/govern-core/contracts/GovernRegistry.sol

```diff
@@ -36,4 +35,11 @@ contract GovernRegistry is IERC3000Registry {
     function _setMetadata(IERC3000Executor _executor, bytes memory _metadata) internal {
         emit SetMetadata(_executor, _metadata);
```

Comment:
Maybe now the metadata should be related to the owner address instead of the executor?

Reply:
Yup, agreed. I thought already before it is strange that anyone can change the metadata of a registered ``Executor``.